buildlib/azure-pipelines-pr.yml

```diff
@@ -139,13 +137,28 @@ stages:
         timeoutInMinutes: 240
 
         steps:
+          - bash: |
+              set -x
+              echo "dir: $(Build.Repository.LocalPath)"
+              lsof $(Build.Repository.LocalPath) || true
+              retry=3
+              while [[ ${retry} != 0 ]]
+              do
+                chmod 755 -R $(Build.Repository.LocalPath)
+                rm -rf $(Build.Repository.LocalPath) && break
+                sleep 5
+                retry=$((retry-1))
+              done
```

Comment:
maybe put this in az-helpers?
do we need it in other stages (e.g coverity, java, ..) ?
BTW, seems like it happens only when job running in docker